src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { BsFacebook, BsGoogle } from 'react-icons/bs';
+
+function SignIn() {
```

Comment:
Because you used react form for signup you can use it to sign in too

Reply:
I think we will be using the authentication here,not use form. If the account exists it will sign in if not it will hive an error msg, so we will need to wait for the auth functionality for this

Reply:
but even for the authentication you need the data from the form, react form is used to handle the states instead of using useState

Reply:
Idk i thought about leaving it till we finish the auth functionality and then we can do the react form here. Otherwise I'll be doing the form the same way I did it in the sign up which is kinda wrong idk.

Reply:
no problem we will fix it in the future

Reply:
Be sure to create an issue to fix this later **if** we wont deal with it in the auth functionality.

Reply:
@Iseluin so should I implement react form here too or should I leave it this way?